controller/src/beerocks/master/tasks/optimal_path_task.cpp

```diff
@@ -84,15 +84,14 @@ void optimal_path_task::work()
     case START: {
 
         current_hostap_vap = database.get_node_parent(sta_mac);
-
-        auto current_vap_name = database.get_hostap_iface_name(current_hostap_vap);
-        current_vap_name.append(".");
-        current_vap_name.append(std::to_string(database.get_node_vap_id(sta_mac)));
-        auto steer_vaps = database.config.load_steer_on_vaps;
-        if (steer_vaps.find(current_vap_name) == std::string::npos) {
-            TASK_LOG(INFO) << "current hostap vap:" << current_vap_name << " mac:" << sta_mac
-                           << " is not in steer list:" << steer_vaps << " aborting optimal task"
-                           << std::endl;
+        // Steering allowed on all vaps unless load_steer_on_vaps list is defined
+        // on the platform , in that case verify that vap is on that list
+        if (!database.config.load_steer_on_vaps.empty() &&
+            !database.is_vap_on_steer_list(current_hostap_vap)) {
+            TASK_LOG(WARNING) << "client " << sta_mac << " is connected to vap "
+                              << current_hostap_vap << " that is currently not in steer list: "
+                              << database.config.load_steer_on_vaps << " aborting optimal task"
+                              << std::endl;
```

Comment:
Why did you add an `endl`? Just curious.

Reply:
copy paste. removed